common/scala/src/main/resources/application.conf

```diff
@@ -197,6 +209,11 @@ whisk {
                 retention-bytes = 1073741824
                 retention-ms    = 3600000
             }
+            actions {
```

Comment:
Yes, this configuration is for topics, `schedulerN` and these topics are used to forward activations from controllers to schedulers.
I feel it would be better to rename it to something like `activations`.